### ISSUE TITLE
Patch 7.3 Update

### ISSUE THREAD

**Issue by dsblack115** [nmsmith22389/HotLoot#25]
I would post an error screenshot, but I am receiving a bunch of UI errors from various addons since Patch 7.3 was just released today.  The HotLoot addon does not hide the anchor, and when trying to select which window to display loot messages, it immediately errors upon clicking the drop down arrow.

**Comment by dsblack115** [comment]
Update: HotLoot is working fine.  It was another addon causing it to not function at all.  Will need to be updated for 7.3, I believe there is another currency option with the addition of Argus.  I have not gotten to that point yet.

_Side note: Thank you for your continued support on this addon.  Hopefully sometime in the future I will be able to compensate via monetary donation for your efforts.  I really appreciate you coming back when I emailed you a while back!_

**Comment by dsblack115** [comment]
With Skinning Mode enabled, it deleted every piece of Stonehide Leather in my bags. (I had just enabled it before this happened)
![wowscrnshot_091017_184009](https://user-images.githubusercontent.com/22847171/30253678-dd26b10a-9657-11e7-9dec-b2f0c7f3e01d.jpg)

**Comment by nmsmith22389** [comment]
Hey! I was actually just about to release an update! Thanks for bringing this one up... I'll add it before I post the update. Thanks!

PS. Sorry about your items 😭

**Comment by nmsmith22389** [comment]
One quick question... when it deleted that leather was it being caught in the loot filter?

What I am think happened is...
1) The leather wasn't caught in the filter.
2) HL deletes stuff not caught in filter by actually looting it then deleting it from your bags.
3) HL deleted the whole stack of leather because it thought you didn't need it.
*(right now in skinning mode when HL goes to delete an item it deletes **ALL** of it from your bags... I'm not sure if this needs to be fixed or not... my thinking was you wouldn't want the item anyways so get rid of all of it.)*

I don't know if that's actually what happened... I'll have to do some more investigating. Let me know if anything pops up. Also you should be able to test if an item would be looted in the filters by typing `/hl filter [item]`

**Comment by dsblack115** [comment]
I’m unsure.  It’s how I normally have it set for whatever professions I have, in the case of my rogue, skinning.  So I have it set to loot leather and skinning mode to delete leftover loot from the beast to enable skinning it without looting junk that I don’t need.

**Comment by nmsmith22389** [comment]
Hmmmm I wonder why it would think Stonehide Leather is junk... I'm kinda stumped on this one... I've used it with skinning before and that didn't happen. I'll have to do some more testing.

Did it do this just that one time then go back to working?

**Comment by dsblack115** [comment]
I haven’t activated that function since it happened.  I turned it on, killed and skinned something once, and it happened.  It didn’t just delete Stonehide, it also deleted unbroken tooth, and claw.  I was able to retrieve them by submitting a blizzard ticket, but I don’t want to risk over 2700 items getting deleted again.

**Comment by dsblack115** [comment]
If you need me to test it again, I can put most of my stacks in bank bags, and test it with a small stack in my regular bags.  Won’t be able to do that until later when I get home from work.

**Comment by dsblack115** [comment]
Also, maybe an update for 7.3 compatibility might fix it.  If you’d like me to wait for you to push the next update.  Up to you.  I’m not privy to coding.

**Comment by dsblack115** [comment]
![capture 3](https://user-images.githubusercontent.com/22847171/30773268-7d2cbb4e-a03b-11e7-8f07-c80f494a38c7.PNG)

Updated to the latest version of HotLoot this morning, tested skinning with skinning mode on, still having the issue.  It is not just Stonehide/Unbroken Tooth and Claw it deletes, I believe it will delete any type of leather in the bags when skinning mode is on as you can see it also deleted Stormscale.  What can I do to help?

Edit: It does not delete Unbroken Claw or Tooth now, I noticed they were still in my bags.

**Comment by nmsmith22389** [comment]
tomorrow im going to do some testing and see if I can figure out why it would do that... by default it shouldn't ever delete anything caught in the filters... I'm gonna go over the skinning mode code and see if I can find anything. Let me know if you find anything out!

**Comment by dsblack115** [comment]
I don't know if this is a bug or not, or should a separate issue be created, but I have noticed the toast is inconsistent, sometimes the toast will show, most of the time it won't show when looting items.  Can't really capture this issue via screenshot, and no error occurs.

**Comment by nmsmith22389** [comment]
About this new issue if you could open a new issue that would be great! Also, any more detail or steps to reproduce it that you could add would be nice. Have you found out anything else about the skinning mode problem? Is it still happening? I couldn't reproduce it myself. The only thing I can think of is either rewriting the skinning mode function and/or adding something like a 'safe list' where it will only delete items on that list or something. Let me know if you have any good ideas! Thanks!

**Comment by dsblack115** [comment]
Toast issue seems fine now.  Must've been a setting on my end.  Last time I tested with the latest update, it deleted Stormscale, I used a small stack to test it.

**Comment by dsblack115** [comment]
Started up a new character with skinning, with skinning mode on, works as it is supposed to with low level mats.

**Comment by nmsmith22389** [comment]
Yeah I'm starting to wonder if at one point a setting got messed up on a character *or even the sell list itself*.

I released an alpha version of a big change to HotLoot that completely changed the way you use filters as well as adding custom sell filters. I would give that a try with brand new profiles. Also I could really use your input about the alpha version in general! :smile:
